README.md

```diff
@@ -1,2 +1,65 @@
 # 1st-Project-YoutubeSite
-1st HTML/CSS project.
+# Embedded HTML/CSS project. :grinning: :+1:
```

Comment:
Emojis are not necessary for project title. Always keep your readme professional as much as you can. Kindly remove the emojis.